src/tools/rustfmt/src/expr.rs

```diff
@@ -137,6 +137,9 @@ pub(crate) fn format_expr(
         ast::ExprKind::Tup(ref items) => {
             rewrite_tuple(context, items.iter(), expr.span, shape, items.len() == 1)
         }
+        ast::ExprKind::Use(_, _) => {
+            unimplemented!();
```

Comment:
Might be better to return the code unformatted instead of panicking:

```suggestion
            Ok(context.snippet(expr.span()).to_owned());;
```

If possible, could we add a test case to `src/tools/rustfmt/tests/target/` annotated with `#![feature(ergonomic_clones)]` that demonstrates the proposed syntax.

Reply:
Yes, I intended to address this later but I'm going to do what you're suggesting and add a FIXME.

Reply:
Awesome, thanks!